docs/tsc/meetings/2020-07-20.md

```diff
@@ -0,0 +1,110 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+July 20, 2020
+
+Host: Michael Dolan
+
+Attendees:
+  * [ ] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [X] Sean Cooper (_TSC ACES TAC Rep_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [ ] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [ ] Carol Payne (_TSC_) - Netflix
+  * [X] Mark Titchener (_TSC_) - Foundry
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] Troy Sobotka
+  * [X] Joseph Goldstone - ARRI
+  * [X] Chin-Ying Li
+  * [X] Simran Brucherseifer - ArangoDB
```

Comment:
I'm here in my spare time, no company involvement
```suggestion
  * [X] Simran Brucherseifer
```

Reply:
Fixed.